lib/ex_aws/ses.ex

```diff
@@ -169,9 +169,9 @@ defmodule ExAws.SES do
       |> Map.merge(format_member_attribute(:reply_to_addresses, opts[:reply_to]))
       |> Map.merge(format_tags(opts[:tags]))
       |> Map.merge(format_dst(dst))
-      |> Map.put_new("Source", src)
-      |> Map.put_new("Template", template)
-      |> Map.put_new("TemplateData", template_data)
+      |> Map.put("Source", src)
+      |> Map.put("Template", template)
+      |> Map.put("TemplateData", format_template_data(template_data))
```

Comment:
Why did you change all `put_new` for `put`?

Reply:
Because the value has to be replace if already exist. For exemple: `TemplateData` was not replaced by `format_template_data(template_data)` because it was already in the map